sympy/core/tests/test_sympify.py

```diff
@@ -1,6 +1,6 @@
 from sympy import (Symbol, exp, Integer, Float, sin, cos, log, Poly, Lambda,
     Function, I, S, sqrt, srepr, Rational, Tuple, Matrix, Interval, Add, Mul,
-    Pow, Or, true, false, Abs, pi, Range, Xor)
+    Pow, Or, true, false, Abs, pi, Range, Xor, FiniteSet)
```

Comment:
I'm not the person to be reviewing these things but I'll suggest why the builds are failing. After the tests start passing for your PR, more people may review it.

I suggest using an editor that identifies unused imports and double imports. It should also identify when a variable cannot be found. This PR is failing because it has a double import of `FiniteSet` in `sympy/core/tests/test_sympify.py`. Your other PR is failing because in `sympy/testing/runtests.py`, it uses `print_function` and `division`. There are probably a lot of other similar errors in that PR.

Instead of having to push your commit to check if it passes the tests, you are able to do so on your local machine. See [this](https://docs.sympy.org/latest/modules/testing/runtests.html) for more info.